<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentInfo</class>
 <widget class="QWidget" name="StudentInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Программа</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="2">
    <widget class="QPushButton" name="pb_save">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QTableWidget" name="tw_contactData"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Имя</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QTableWidget" name="tw_paymentHistory"/>
   </item>
   <item row="9" column="2" colspan="2">
    <widget class="QPushButton" name="pb_discard">
     <property name="text">
      <string>Отменить</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QSpinBox" name="sb_grossPayment">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Прогресс</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QPushButton" name="pb_addPayment">
     <property name="text">
      <string>Добавить оплату</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLineEdit" name="le_name"/>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QLineEdit" name="le_educationalPlan"/>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QPushButton" name="pb_editStudent">
     <property name="text">
      <string>Добавить/изменить</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Баланс, руб.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="le_educationalPlanProgress"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
